target/chatapp-1.0-SNAPSHOT/about.jsp

```diff
@@ -0,0 +1,73 @@
+<%--
```

Comment:
I think this is meant to be changed in src/main/webapp/about.jsp rather than target/...

Can you change the file you've changed to the one in src/, so I can see the diffs?

Reply:
got it!

Reply:
Please remove this file now:
git rm <filename>
git commit
git push